src/components/procedure/_platform-detect-ghrepos.md

```diff
@@ -0,0 +1,8 @@
+To ensure that your GitHub repository is **detected** by Semgrep Cloud Platform:
+
+1. Log into GitHub.
+2. Click your **profile photo > Settings > Applications**.
```

Comment:
I don't think this set of steps will work for organizations as you need to select your org first. What would be the best way for us to handle that in this snippet?

Reply:
Nice catch, I created separate steps for personal vs org users :)